components/bsx/Gallery/Item/GalleryItem.vue

```diff
@@ -81,7 +81,7 @@
                           {{ $t('Unlist') }}
                         </b-button>
                       </div>
-                      <div v-if="nftRoyalties">
```

Comment:
I wonder why you remove this check? What if a nft has no royalties

Reply:
yeah my bad, must have been removed accidentally, added it again, thank you for pointing it out